### DISCUSSION TITLE
Lesson 19 On Udemy is repeated

### DISCUSSION THREAD

**Discussion by Madraceee** [Cyfrin/foundry-full-course-cu#373] [Q&A]
I am not sure whether it is udemy's fault or uploader's fault, but lesson 18 and 19 on udemy are repeated. I am not sure if others faced the same issue. Hope it is resolved.
Thank you for the course :)

Edit: Udemy videos 18 , 19

**Comment by PatrickAlphaC** [comment]
Thanks! Udemy kicked me off sadly :(
